src/index.html

```diff
@@ -1,13 +1,319 @@
+                      alt="Portable speakers 2"
+                    >
+                  </a>
+                </div>
+
+                <h3 class="category__title">
+                  Portable speakers
+                </h3>
+              </article>
+
+              <article class="category__product">
+                <div class="category__gallery">
+
+                  <a href="#" class="category__link">
+                    <img
+                      src="./images/e1.png"
+                      class="product__photo"
+                      alt="Earphones 1"
+                    >
+                  </a>
+
+                  <a href="#" class="category__link">
+                    <img
+                      src="./images/e2.png"
+                      class="product__photo"
+                      alt="Earphones 2"
+                    >
+                  </a>
+                </div>
+
+                <h3 class="category__title">
+                  Earphones
+                </h3>
+              </article>
+
+              <article class="category__product">
+                <div class="category__gallery">
+
+                  <a href="#" class="category__link">
+                    <img
+                      src="./images/sh1.png"
+                      class="product__photo"
+                      alt="Smart Home 1"
+                    >
+                  </a>
+
+                  <a href="#" class="category__link">
+                    <img
+                      src="./images/sh2.png"
+                      class="product__photo"
+                      alt="Smart Home"
+                    >
+                  </a>
+
+                </div>
+
+                <h3 class="category__title">
+                  Smart Home
+                </h3>
+              </article>
+
+              <div class="category__block">
+                <a href="#" class="category__button section-button">View all</a>
+              </div>
+            </div>
+          </section>
+        </div>
+
+        <section id="about-us" class="about-us">
+          <div class="about-us__background"></div>
+          <div class="about-us__information">
+            <h2 class="section-title about-us__title">Timeless, for 50 years and counting</h2>
+
+            <p class="about-us__text">
+              Bang & Olufsen is expanding its Recreated Classics Program with the launch of Beosystems – a limited edition music system that transcends time by bridging the gap between one of our iconic designs from 1972 and today’s cutting-edge digital technology.
+            </p>
+          </div>
+        </section>
+      </div>
+    </main>
+
+    <footer>
+      <section id="contact-us" class="contact-us">
+        <div class="container">
+          <h2 class="section-title">Contact us</h2>
+          <div class="contact-us__content">
+            <form action="#" class="contact-us__form form">
+
+              <label for="name">
+                <input
```

Comment:
11. [CHECKLIST] All form fields should be required and have correct types (email, tel, etc.)